README

```diff
@@ -1,14 +1,14 @@
-WiredTiger 10.0.2: (November 30, 2021)
+WiredTiger 11.0.0: (May 12, 2022)
 
-This is version 10.0.2 of WiredTiger.
+This is version 11.0.0 of WiredTiger.
 
 WiredTiger release packages and documentation can be found at:
 
 	https://source.wiredtiger.com/
 
 The documentation for this specific release can be found at:
 
-	https://source.wiredtiger.com/develop/index.html
+	https://source.wiredtiger.com/11.0.0/index.html
```

Comment:
This change is a bit annoying, we need to for the new release on master branch, but need to revert the change on develop branch afterward. I'd suggest we simply put the website homepage URL (https://source.wiredtiger.com) here instead.

Reply:
Why do we need to revert?

I don't object, but I don't understand either.

Reply:
Currently, the master branch (where all OSS releases are tagged) basically takes a merge from develop branch every time an OSS release needs to be cut. We try to avoid diverging master from develop with the link change like above (develop--> 11.0.0), which means the link change needs to be made on develop first, merged/sync-ed to master, and reverted on develop. 

My suggestion is to replace the release-specific doc link with the common doc URL https://source.wiredtiger.com/) in README, where users can easily find the release-specific doc link.